tests/monotouch-test/Security/CertificateTest.cs

```diff
@@ -555,7 +564,12 @@ public void MailX1 ()
 		[Ignore ("https://bugzilla.xamarin.com/show_bug.cgi?id=39952")]
 		public void MailX2 ()
 		{
-			using (var cert = new X509Certificate2 (mail_google_com)) {
+#if MONOMAC // Not disposable on mac
```

Comment:
Apparently the xammac_tests.csproj was the wrong project type (and as a result was using system mono).  Fixed that, have it targeting XM mobile and it's working fine now.